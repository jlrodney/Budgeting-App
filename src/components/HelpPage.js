import React from 'react';

const HelpPage = () => (
  <div>
    Please contact James Rodney with any problems at jlrodney95@gmail.com
  </div>
);

export default HelpPage;
