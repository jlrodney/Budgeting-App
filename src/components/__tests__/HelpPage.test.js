import React from 'react';
import { shallow } from 'enzyme';
import HelpPage from '../HelpPage';

describe('HelpPage should render  correctly', () => {
  it('should have a div', () => {
    const wrapper = shallow(<HelpPage />);
    expect(wrapper.find('div').text()).toEqual("This is the help page");
  })
});

describe('should render Header correctly', () => {
  const wrapper = shallow(<HelpPage />);
  expect(wrapper).toMatchSnapshot();
})
