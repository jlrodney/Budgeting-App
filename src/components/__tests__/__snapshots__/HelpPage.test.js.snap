// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  This is the help page
</div>
`;
